<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remoteview</class>
 <widget class="QWidget" name="remoteview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>597</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SFTP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButton_6">
        <property name="toolTip">
         <string>Bookmarks</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="nullfxp.qrc">
          <normaloff>:/icons/folder-favorites.png</normaloff>:/icons/folder-favorites.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>ixvxcv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>vxc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>oiew</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dsfsdf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clear...</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_5">
        <property name="toolTip">
         <string>Go Home</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="nullfxp.qrc">
          <normaloff>:/icons/user-home.png</normaloff>:/icons/user-home.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_4">
        <property name="toolTip">
         <string>Go Previous</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="nullfxp.qrc">
          <normaloff>:/icons/go-previous.png</normaloff>:/icons/go-previous.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_3">
        <property name="toolTip">
         <string>Go Next</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="nullfxp.qrc">
          <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="toolTip">
         <string>Go Parent</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="nullfxp.qrc">
          <normaloff>:/icons/go-up.png</normaloff>:/icons/go-up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="toolTip">
         <string>Reload</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="nullfxp.qrc">
          <normaloff>:/icons/appointment-recurring.png</normaloff>:/icons/appointment-recurring.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="RemoteViewDirTableView" name="tableView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RemoteViewDirTableView</class>
   <extends>QTableView</extends>
   <header location="global">remoteviewdirtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="nullfxp.qrc"/>
 </resources>
 <connections/>
</ui>
